.github/workflows/valkey.yml

```
@@ -13,33 +13,41 @@ concurrency:
+
+env:
+  CLI_COMMAND: "valkey-cli"
+  IMAGE_NAME: "valkey/valkey"  # Use the official Valkey image
+
+
 jobs:
   valkey-test:
     # Descriptive name for the job runs in the UI
-    name: Py${{ matrix.python-version }}/Valkey${{ matrix.valkey-version }}/redis-py${{ matrix.redis-py-version }}
+    name:
+      "Py${{ matrix.python-version }}/Valkey${{ matrix.valkey-version }}\
```

actionlint (1.7.12)

[error] property "valkey-variant" is not defined in object type {python-version: number; redis-py-version: number; valkey-version: number; variant: string}

(expression)